packaging/install.cmake

```diff
@@ -67,60 +67,64 @@ endif()
 
 if(UNIX)
 
-  # replace MPIEXEC MPIEXEC_NUMPROC_FLAG and MPI_IMPL KHIOPS_MPI_EXTRA_FLAG ADDITIONAL_EN_VAR
+  # Set khiops and khiops_coclustering paths according to the environment (conda, fedora, etc)
+  if(IS_CONDA)
+    set(KHIOPS_PATH "$(get_script_dir)")
+    set(KHIOPS_COCLUSTERING_PATH "$(get_script_dir)")
+    set(GET_PROC_NUMBER_PATH "$(get_script_dir)")
+  else()
+    if(IS_FEDORA_LIKE)
+      set(KHIOPS_PATH "${MPI_BIN}/khiops/")
+    else()
+      set(KHIOPS_PATH "/usr/bin/")
+    endif(IS_FEDORA_LIKE)
+    set(KHIOPS_COCLUSTERING_PATH "/usr/bin/")
+    set(GET_PROC_NUMBER_PATH "/usr/bin/")
```

Comment:
Hence, on Rocky, `MODL` and `MODL_Coclustering` will have different paths on native installations; I wonder if putting both executables in the same path (albeit prefixed with the MPI library) would be clearer to the end-users or integrators.